compiler/modules/types/core.bal

```diff
@@ -171,7 +172,10 @@ function cellAtomType(Atom atom) returns CellAtomicType {
 }
 
 // See memoSubtypeIsEmpty for what these mean.
```

Comment:
Adding the comment here is not consistent with this line.

Reply:
fixed